### PR TITLE
[SparklineChart] aria role fix for <path/>

### LINKED ISSUE #29936: [Bug]: Sparkline <path> element does not have associated role (aria-allowed-attr)
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (20) x64 13th Gen Intel(R) Core(TM) i7-13800H
    Memory: 35.07 GB / 63.66 GB
  Browsers:
    Edge: Chromium (119.0.2151.93)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/sparkline

### Bug Description

We are leveraging `@fluentui/react-charting`'s `Sparkline` and `GaugeChart` components for a few of our react blades. As part of release readiness, we run the Accessibility Insights for Web's FastPass automated checker. We noticed that in the Sparkline component, the [aria-allowed-attr ](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr/) check is failing. the <path> elements contain the aria-label but no associated role. This can  be reproduced by going to the provided examples and running FastPass on the `Sparkline `example:

sparkline_fastpass

## Actual Behavior
The FastPass automated check fail due to the <path> element containing an aria-label attribute with no valid role attribute. 
## Expected Behavior
The associated <path> element should have a corresponding role attribute. The FastPass automated checks should pass.

### Logs

_No response_

### Requested priority

High

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior
ARIA violations on sparklines
image

## New Behavior
ARIA violations on sparkline chart fixed
image

## Related Issue(s)

- Fixes #29936

### CODE REVIEW

File: change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added img aria role to <path/>",
```
Comment (AtishayMsft) [inline change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json:3]: Update comment to include entry for SparklineChart

Comment (Shubhabrata08) [reply change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json:3]: Ok

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2ca8d35248eded0e15e0a342c4ed8d7b75601c17'>2ca8d35248eded0e15e0a342c4ed8d7b75601c17</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30155&commitId=7d5260379c32e027206d3e71c4eb984b11b5bdbe&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (AtishayMsft) [reply change/@fluentui-react-charting-d420d655-3051-4fef-9dba-d672cec9ee6f.json:1]: Unrelated change

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**